examples/manage/manage-environment/Jenkinsfile

```
@@ -147,6 +147,7 @@ pipeline {
         stage('Manage Env') {
             steps {
                 sh '''#!/bin/bash
+                    ${GENERATION_DIR}/setup.sh
```

This is available in the CLI now

```suggestion
                    hamlet setup
```

---

Agreement is to leave script until such time as we are ready to cut across all interaction with hamlet to the cli.